spec/rails_helper.rb

```diff
@@ -7,6 +7,11 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
+require 'webmock/rspec'
+WebMock.enable_net_connect!
+
```

Comment:
Your use of webmock looks good!

Usually as part of the webmock configuration, I add `WebMock.disable_net_connect!(allow_localhost: true)` so that Solr requests, etc can go through. That's probably not needed for this app, but just wanted to let you know just in case.

Reply:
eventually, i'd like to get to the point where i can disable all connections for this app, but that will take a while :sun_with_face: